packages/core/demo/data/confidence-interval.ts

```diff
@@ -0,0 +1,61 @@
+export const confidenceIntervalTimeSeriesData = [
+	{ group: "Dataset 1", date: new Date(2019, 0, 1), value: 50000, min: 30000, max: 60000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 5), value: 15000, min:5000, max: 30000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 8), value: 20000, min: 10000, max: 35000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 13), value: 39213, min: 10000, max: 45000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 19), value: 61213, min: 50000, max: 70000 }
+];
+
+export const confidenceIntervalTimeSeriesOptions = {
+	title: "Confidence Interval (time series)",
+	legend: {
+		enabled: false
+	},
+	confidence : {
+		upperBoundMapsTo: "max",
+		lowerBoundMapsTo: "min" 
+   	},
```

Comment:
Is there a reason this couldn't be made generic for any axis chart? It's not uncommon to see error bars on scatter plots or bar charts .... I don't entirely understand why it needs to be a whole new chart?

Reply:
@cal-smith I think adding it directly to line chart would work, but in terms of adding it to all axis charts Im not sure if this would work for all chart types. For example area chart is also an axis chart but further adding a confidence interval on to this would get messy. In this case error bars would be needed as an alternate to confidence interval. Which is why i think @theiliad suggested separating out the confidence behaviour to be used for future graphs.

Reply:
I was wondering this as well, is it not better suited to be an option for an area chart to be bounded with min/max values **or we have a carbon-charts `BoundedAreaChart`** similar to stacked area but then combine it with a `Line` chart (or multiple line charts) via `ComboChart`? 

FYI the bounded area + line issue is here:
https://github.com/carbon-design-system/carbon-charts/issues/723

Reply:
@natashadecoste a `BoundedAreaChart` sounds good! I think that is essentially what is happening now as the confidence interval component is currently a bounded area and the line is just drawn on by calling the line chart component (in a similar way to what the area chart is currently doing). But yeah making it an additional option for the area chart to be bounded or making a `BoundedAreaChart` similar to stacked i think might be better then having it available to all axes type charts

Reply:
Interesting ... We definitely don't have to add it to everything _now_, but as long as we have the behaviour for confidence intervals/error bars in a central spot, and we can add it as needed to _existing_ chart types we should be good. It seems like the main barrier to adding it to more axis/Cartesian charts is primarily design related, so I'd still want to make sure that we aren't coding ourselves into a corner.

@natashadecoste's suggestion seems reasonable as well if we need a way to expose this as a first pass. I'd probably lean towards making it a config, but that's totally up to You @shaziajk, @theiliad, and @natashadecoste

Reply:
So maybe then `confidence` needs to also be changed to `bounds` => { upper, lower }

Reply:
the reason we wouldn't want to make it just a config is that a bar chart for instance wouldn't really benefit from the bounds showing up behind the bars

Reply:
ok sounds cool would it make more sense to follow the making option for an area chart to be bounded with min/max values suggested by @natashadecoste and then chance confidence to bounds => { upper, lower }? 
rather then make a whole `BoundedAreaChart` which follow a similar approach to stacked?